sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IOReadIT.java

```diff
@@ -86,36 +86,6 @@ public void tearDown() throws Exception {
     deleteAllHL7v2Messages(this.client, healthcareDataset + "/hl7V2Stores/" + HL7V2_STORE_NAME);
   }
 
-  @Test
```

Comment:
these tests are actually redundant with the non-deleted `testHL7v2IO_ListHL7v2Messages` and `testHL7v2IO_ListHL7v2Messages_filtered`